libs/types/src/tokens.rs

```diff
@@ -364,6 +364,23 @@ impl CrossChainTransferability {
 	pub fn includes_liquidity_pools(self) -> bool {
 		matches!(self, Self::LiquidityPools)
 	}
+
+	/// Fees will be charged using `FixedRateOfFungible`.
+	#[cfg(feature = "std")]
+	pub fn xcm_default() -> Self {
+		Self::Xcm(XcmMetadata {
+			fee_per_second: None,
+		})
+	}
+
+	/// Fees will be charged using `AssetRegistryTrader`.
+	/// If value is 0, no fees will be charged.
+	#[cfg(feature = "std")]
+	pub fn xcm_with_fees(value: Balance) -> Self {
+		Self::Xcm(XcmMetadata {
+			fee_per_second: Some(value),
+		})
+	}
```

Comment:
Tech-debt to create some tests to be able to compute the exact fee charged in the above two cases. Using `FixedRateOfFungible` and using `AssetRegistryTrader`